drivers/i2c/Kconfig.nrfx

```diff
@@ -0,0 +1,115 @@
+# Kconfig - nrfx I2C support
+#
+# Copyright (c) 2017 - 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig I2C_NRFX
+	bool "nRF TWI nrfx drivers"
+	depends on SOC_FAMILY_NRF
+	help
+	  Enable support for nrfx TWI drivers for nRF MCU series.
+	  Peripherals with the same instance ID cannot be used together,
+	  e.g. I2C_0 and SPI_0. You may need to disable SPI_0 or SPI_1.
+
+if I2C_NRFX
+
+config I2C_NRFX_PINS_OVERRIDE_DTS
+	bool "Override I2C pin settings from DTS"
+	default n
+	help
+	  Allows setting of SDA and SCL pins using menuconfig.
+	  Use when connecting devices to development kits without
+	  specified I2C pins.
+
+# Nordic TWIx0 and SPIx0 instances can not be used at the same time.
+if I2C_0 && !SPI_0
+
+choice I2C_0_NRF_TYPE
+	prompt "I2C Port 0 Driver type"
+
+config I2C_0_NRF_TWI
+	bool "nRF TWI 0"
+	depends on SOC_SERIES_NRF52X || SOC_SERIES_NRF51X
+	select NRFX_TWI
+	help
+	  Enable nRF TWI Master without EasyDMA on port 0.
+
+config I2C_0_NRF_TWIM
+	bool "nRF TWIM 0"
+	depends on SOC_SERIES_NRF52X
+	select NRFX_TWIM
+	help
+	  Enable nRF TWI Master with EasyDMA on port 0.
```

Comment:
It would be good to mention here that this peripheral accepts only transfers from RAM, and when a buffer placed in flash is provided, the transfer will fail.

Reply:
Done